javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,131 @@
+let info = [{       // User name array
+    name: ""
+}]
+
+let todolist = []       // Todo list array
+
+function getReply(command) {
+    let arr = []        // Users inputs arrays
+    command1 = command.split(' ')
+    arr.push(command1)
+
+    switch (arr[0][0] + arr[0][1] + arr[0][2] + arr[0][3]) {
+
+        // Introduce inputs
+
+        case 'Hellomynameis':
```

Comment:
If you decided to use `.toLowerCase` earlier, this check would always be:
`case 'hellomynameis'`